### DISCUSSION TITLE
can i add my profile to the  all contributors list in the readme?

### DISCUSSION THREAD

**Discussion by HimangshuSharma01** [ML-Fusion-Lab/ML-Fusion-Lab-Website#888] [General]
I have contributed to this project can i add my profile in the readme page? @vivekvardhan2810

**Comment by vivekvardhan2810** [comment]
@HimangshuSharma01 for now it is no needed, 

As  i can see your photo in readme.

![image](https://github.com/user-attachments/assets/22caf591-13b7-47fe-81ee-bcdbeaf620b1)

**Reply by HimangshuSharma01** [reply]
im sorry it was not loading
